src/App.tsx

```diff
@@ -1,16 +1,13 @@
 import "./App.scss";
 import { Link, Outlet } from "react-router-dom";
 import paths from "./Paths/paths";
+import NetComponent from "./Components/FirstScreen/Net-component/Net";
```

Comment:
A few ideas
1) in order to avoid importing from such a long path you can use index files.
Meaning, create index.ts in the folder (e.g. Net-component) and add export * from "./Net"
It will allow you to import from "./Components/FirstScreen/Net-component".

If you do this on other folders as well, you can achieve import  from "./Components" . It may be useful.
2) currently you use relative imports. If you will do restructuring of files, move them arounf, all paths will have to be adjusted. You can implement so called "aliases" . Check out this article https://medium.com/@devxprite/how-to-setup-path-aliases-in-vite-df955939ffe8